bpf/init.sh

```diff
@@ -196,62 +196,6 @@ function rnd_mac_addr()
     printf '%02x%s' $upper $lower
 }
 
-function bpf_compile()
-{
-	IN=$1
-	OUT=$2
-	TYPE=$3
-	EXTRA_OPTS=$4
-
-	clang -O2 -target bpf -std=gnu89 -nostdinc -emit-llvm	\
-	      -g -Wall -Wextra -Werror -Wshadow			\
-	      -Wno-address-of-packed-member			\
-	      -Wno-unknown-warning-option			\
-	      -Wno-gnu-variable-sized-type-not-at-end		\
-	      -Wdeclaration-after-statement			\
-	      -Wimplicit-int-conversion -Wenum-conversion	\
-	      -I. -I$DIR -I$LIB -I$LIB/include			\
-	      -D__NR_CPUS__=$NR_CPUS				\
```

Comment:
Commenting here as I can't comment on untouched lines. Reminder to check for unused script input args. `ID_WORLD` can also be removed.